jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/core/JakartaForJavaAssert.java

```diff
@@ -156,22 +156,32 @@ public static Range r(int startLine, int startCharacter, int endLine, int endCha
     public static Position p(int line, int character) {
         return new Position(line, character);
     }
+  
+    public static void assertJavaDiagnosticsPresent(JakartaDiagnosticsParams params, JDTUtils utils, Diagnostic... expected)
```

Comment:
Since this method is going to assert a diagnostic is not present it should be renamed assertJavaDiagnosticsNotPresent().